### PR TITLE
[TSC] Update CODEOWNERS

### PR DESCRIPTION
**Pull request by walterkozlowski**
Added @walterkozlowski as a code owner

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -34,75 +34,75 @@
...
+/doc/ref_cert/RC2/chapters/chapter08.md        @xmulligan @scottsteinbrueck @walterkozlowski
 /doc/ref_cert/RC2/chapters/chapter09.md        @xmulligan @scottsteinbrueck
 
 # Edge
-/doc/tech/usecases.md        @ASawwaf @scottsteinbrueck
+/doc/tech/usecases.md        @ASawwaf @scottsteinbrueck @walterkozlowski
 
 # NFG
-/doc/tech/approach.md        @walterkozlowski @scottsteinbrueck
+/doc/tech/approach.md        @walterkozlowski @scottsteinbrueck @walterkozlowski
```
Comment (pgoyal01) [inline CODEOWNERS:108]: @walterkozlowski  you are listed twice

File: CODEOWNERS [hunk 2]
```diff
@@ -34,75 +34,75 @@
...
-/doc/ref_cert/RC2/chapters/chapter08.md        @xmulligan @scottsteinbrueck
+/doc/ref_cert/RC2/chapters/chapter01.md        @xmulligan @scottsteinbrueck @walterkozlowski
+/doc/ref_cert/RC2/chapters/chapter02.md        @xmulligan @scottsteinbrueck @walterkozlowski
+/doc/ref_cert/RC2/chapters/chapter03.md        @xmulligan @scottsteinbrueck @walterkozlowski
+/doc/ref_cert/RC2/chapters/chapter05.md        @xmulligan @scottsteinbrueck @walterkozlowski
+/doc/ref_cert/RC2/chapters/chapter04.md        @xmulligan @scottsteinbrueck @walterkozlowski
+/doc/ref_cert/RC2/chapters/chapter06.md        @xmulligan @scottsteinbrueck @walterkozlowski
+/doc/ref_cert/RC2/chapters/chapter07.md        @xmulligan @scottsteinbrueck @walterkozlowski
+/doc/ref_cert/RC2/chapters/chapter08.md        @xmulligan @scottsteinbrueck @walterkozlowski
 /doc/ref_cert/RC2/chapters/chapter09.md        @xmulligan @scottsteinbrueck
```
Comment (pgoyal01) [inline CODEOWNERS:102]:
```suggestion
/doc/ref_cert/RC2/chapters/chapter09.md        @xmulligan @scottsteinbrueck @walterkozlowski 
```

File: CODEOWNERS [hunk 3]
```diff
@@ -34,75 +34,75 @@
...
+/doc/ref_arch/kubernetes/chapters/chapter02.md        @scottsteinbrueck @rgstori @walterkozlowski
+/doc/ref_arch/kubernetes/chapters/chapter03.md        @scottsteinbrueck @rgstori @walterkozlowski
+/doc/ref_arch/kubernetes/chapters/chapter04.md        @scottsteinbrueck @rgstori @walterkozlowski
+/doc/ref_arch/kubernetes/chapters/chapter05.md        @scottsteinbrueck @rgstori @walterkozlowski
+/doc/ref_arch/kubernetes/chapters/chapter06.md        @scottsteinbrueck @rgstori @walterkozlowski
+/doc/ref_arch/kubernetes/chapters/chapter07.md        @scottsteinbrueck @rgstori @walterkozlowski
+/doc/ref_arch/kubernetes/chapters/chapter08.md        @scottsteinbrueck @rgstori @walterkozlowski
 
 
 # Reference Implementation 2
```
Comment (pgoyal01) [inline CODEOWNERS:82]: @tomkivlin no longer engaged. Should we add both Rihab and James

Comment (tomkivlin) [reply CODEOWNERS:82]: I covered this in #2310

Comment (tomkivlin) [reply CODEOWNERS:82]: And yes, to your question.

**Comment by walterkozlowski** [comment]
@pgoyal01, I have accepted your changes.  Cam you please re-review and approve if you can?
@scottsteinbrueck, @tomkivlin, @CsatariGergely - please review and approve if you can.  This follows from requests for me (as the Co-Chair of TSC) to be able to admin GitHub, when necessary.

File: CODEOWNERS [hunk 4]
```diff
@@ -34,75 +34,75 @@
...
-/doc/ref_impl/cntt-ri2/chapters/chapter01.md        @scottsteinbrueck @tomkivlin
-/doc/ref_impl/cntt-ri2/chapters/chapter02.md        @scottsteinbrueck @tomkivlin
-/doc/ref_impl/cntt-ri2/chapters/chapter03.md        @scottsteinbrueck @tomkivlin
-/doc/ref_impl/cntt-ri2/chapters/chapter05.md        @scottsteinbrueck @tomkivlin
-/doc/ref_impl/cntt-ri2/chapters/chapter04.md        @scottsteinbrueck @tomkivlin
-/doc/ref_impl/cntt-ri2/chapters/chapter06.md        @scottsteinbrueck @tomkivlin
-/doc/ref_impl/cntt-ri2/chapters/chapter07.md        @scottsteinbrueck @tomkivlin
-/doc/ref_impl/cntt-ri2/chapters/chapter08.md        @scottsteinbrueck @tomkivlin
-/doc/ref_impl/cntt-ri2/chapters/chapter09.md        @scottsteinbrueck @tomkivlin
+/doc/ref_impl/cntt-ri2/chapters/chapter01.md        @scottsteinbrueck @tomkivlin @walterkozlowski
```
Comment (pgoyal01) [inline CODEOWNERS:83]: @walterkozlowski In this section tomkivlin should be replaced by @rihabbanday @jgu17

Comment (tomkivlin) [reply CODEOWNERS:83]: @pgoyal01 I've already covered this in your previous comment: https://github.com/cntt-n/CNTT/pull/2311#discussion_r595993185

**Review by tomkivlin — APPROVED** [review]

**Review by collivier — CHANGES_REQUESTED** [review]
+ ri1 +rc1 +rc2

@collivier What is the ask?

**Comment by wmk-admin** [comment]
@pgoyal01 @collivier @CsatariGergely - please re-review after a PR#2310  created by @tomkivlin to remove @tomkivlin name has been merged

**Review by pgoyal01 — APPROVED** [review]

**Review by trevgc — APPROVED** [review]

**Review by wmk-admin — APPROVED** [review]

**Review by ulikleber — APPROVED** [review]

**Comment by collivier** [comment]
@walterkozlowski Sorry not to review on the time you asked. But no my comments have not been addressed as you don't change the owners in all streams. For instance @xmulligan stopped several months ago.

**Pull request merged**